### DISCUSSION TITLE
I'm new to Ignition, where do I start?

### DISCUSSION THREAD

**Discussion by chapupu** [ignitionrobotics/testing#20] [Q&A] [answered]
What's the entry-point for a new user?

**Comment by chapulina** [comment, marked as answer]
Go to:

* https://ignitionrobotics.org/
* Click on DOCS: https://ignitionrobotics.org/docs
* Then Get started: https://ignitionrobotics.org/docs/all/getstarted

Have fun!

**Comment by nkoenig** [comment]
I have a different suggestion

**Comment by tfoote** [comment]
another answer test
